src/components/storage/aws.ts

```diff
@@ -114,27 +91,30 @@ export class AwsEventLogStorage implements EventLogStorage {
     }
 
     async storeEvents(events: any) {
-        const key = 'events/events.csv'
-
-        const allEvent = await this._getObject({key, type: 'csv'})
-
-        events.data.forEach((event) => {
-            allEvent.push([event.time, events.id, event.other, event.type])
-        })
+        let storeEventSuccess = true
+
+        for(let event of events.data) {
```

Comment:
`event` is never reassigned within the `for...of` scope, so it should be a `const` rather than `let`. Very minor issue, but something you can set up your linter to warn you